src/fidesops/ops/migrations/versions/a0e6feb5bdc8_add_consent.py

```diff
@@ -0,0 +1,66 @@
+"""Add consent
+
+Revision ID: a0e6feb5bdc8
+Revises: 7a4f4042091e
+Create Date: 2022-09-12 14:46:05.443579
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a0e6feb5bdc8"
+down_revision = "7a4f4042091e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "consent",
+        sa.Column("id", sa.String(length=255), nullable=False),
+        sa.Column(
+            "created_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=True,
+        ),
+        sa.Column("provided_identity_id", sa.String()),
+        sa.Column("data_use", sa.String(), nullable=True),
+        sa.Column("data_use_description", sa.String(), nullable=True),
+        sa.Column("opt_in", sa.Boolean(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["provided_identity_id"],
+            ["providedidentity.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint("provided_identity_id"),
```

Comment:
New tables are a good opportunity to update the database diagram https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md